### ISSUE TITLE
Unable to change element colors

### ISSUE THREAD

**Issue by AndroidKitKat** [CreatureSurvive/Fabric#7]
As of about 6 hours ago, I am completely unable to change the color of elements on the lock screen. I have tried reinstalling the tweak, respringing, re-installing all the dependencies for the tweak, rejailbreaking, but am still unable

Device: iPhone XS, 12.1
Chimera 1.0.4

Tweak List: https://gist.github.com/AndroidKitKat/346046550b2782c325c9fe2a0418034e

If you need anything from me, please do not hesitate to ask. 

Thanks

**Comment by CreatureSurvive** [comment]
@AndroidKitKat sorry for the late response, I've been busy the last few days. I have not been able to reproduce this same issue exactly as you describe, however I have experienced other issues where colors would revert and would not change until after a respring. I have an update just about ready that has fixed all the coloring issues that I've experienced as well as a ton of other bug fixes and enhancements. I'm hoping to have it released later today.
